source/tests/pt/test_finetune.py

```diff
@@ -30,20 +32,47 @@
     model_dpa1,
     model_dpa2,
     model_se_e2_a,
+    model_zbl,
 )
 
 
 class FinetuneTest:
     def test_finetune_change_energy_bias(self):
         # get model
-        model = get_model(self.model_config)
-        model.fitting_net.bias_atom_e = torch.rand_like(model.fitting_net.bias_atom_e)
-        energy_bias_before = deepcopy(
-            model.fitting_net.bias_atom_e.detach().cpu().numpy().reshape(-1)
-        )
-        bias_atom_e_input = deepcopy(
-            model.fitting_net.bias_atom_e.detach().cpu().numpy().reshape(-1)
-        )
+        if "use_srtab" in self.model_config:
+            model = get_zbl_model(self.model_config)
+        else:
+            model = get_model(self.model_config)
+        if isinstance(model, EnergyModel):
+            model.fitting_net.bias_atom_e = torch.rand_like(
+                model.fitting_net.bias_atom_e
+            )
+            energy_bias_before = deepcopy(
+                model.fitting_net.bias_atom_e.detach().cpu().numpy().reshape(-1)
+            )
+            bias_atom_e_input = deepcopy(
+                model.fitting_net.bias_atom_e.detach().cpu().numpy().reshape(-1)
+            )
+        elif isinstance(model, DPZBLModel):
+            model.dp_model.fitting_net.bias_atom_e = torch.rand_like(
+                model.dp_model.fitting_net.bias_atom_e
+            )
```

Comment:
currently, pairtab. bias_atom_e is in initialized to None, we better initialize it to a proper shape.